### ISSUE TITLE
Storage God Mode = true is persisted on storages after save but icon overlays are not

### ISSUE THREAD

**Issue by altmank** [altmank/CaptainOfCheats#4]
Need to recheck status for icon display on load of game.

**Comment by JamieRCHI** [comment]
The slider for increasing the Unity added is also not remembered. Probably the others as well.
I am thinking it would be easier to change the sliders to a text entry box that I type the number in directly.

Easier to see at a glance what I set. (And I think the value only really needs to be between 0 and 20. Minor thing really.)

I don't know if this is a Unity Issue or not, but I expect to be able to use the scroll wheel on my mouse to adjust sliders, but that doesn't work. (There is a up/down spring switch on my Microsoft Natural Keyboard. I'll post if that works or not. Maybe others would find that useful if I find it works.)

And thanks for keeping active with this trainer mod.

**Comment by altmank** [comment]
In general, I avoided keyed entry just because I didn't want to deal with the validation logic but I do plan to add +1, +5, +25 buttons to increment the sliders as well. 

So far I have also avoided doing any kind of serialization of state required to persist values on saving/loading. The only states that currently get saved are because the dev implementations of the underlying features I am hooking into, already save that state. 

I am also using the existing UI libraries so I don't have much (any?) control over scroll wheel support.
